### DISCUSSION TITLE
Hope to add auto graphic card driver updating

### DISCUSSION THREAD

**Discussion by yelan04** [seerge/g-helper#2115] [Ideas]
I think this program will be better by adding auto-graphic card updating, checking, and updating. It will give us so much more convenience that we do not need to worry about the WhisperMode of Nvidia limiting our gaming performance.

**Comment by seerge** [comment]
@yelan04 app already offers you Update section that lists all drivers available on Asus website.

If you want to use GPU drivers from AMD / Nvidia you should check them directly on their sites as well :)
